Fix Makefile to not try to download and install LuaRocks and busted; use
system-provided busted.
--- a/binding/lua/Makefile
+++ b/binding/lua/Makefile
@@ -61,7 +61,7 @@
 depsclean:
 	rm -rf $(DEPS_DIR)
 
-test: $(BUSTED) $(MPACK)
+test: $(MPACK)
 	$(BUSTED) -o gtest test.lua
 
 valgrind: $(BUSTED) $(MPACK)
